### DISCUSSION TITLE
Update on progress of Bridge for Testnet

### DISCUSSION THREAD

**Discussion by ckartik** [NilFoundation/evm-mina-verification#46] [General]
Hey Nil Foundation! I'm wondering if you've got on insights on the progress of the Mina bridge.

**Comment by nemothenoone** [comment]
Hey @ckartik! 

Yes, there are some. We've achieved full codebase readiness and it now underpasses the audit. Meanwhile in preparation for the production launch we're about to arrange a competitive auxiliary proof generation event so when a bridge launch happens, there would be enough amount of those who generate proofs fast enough for the bridge. We're looking forward to Mina community's participation in this!

**Reply by ckartik** [reply]
Exciting!
